scripts/generate-framework-agreement-counterpart-signature-pages.py

```diff
@@ -58,8 +61,10 @@
     args = docopt(__doc__)
 
     framework_slug = args['<framework_slug>']
-    framework_kwargs = declaration_definite_pass_schema = json.load(open(args["<framework_json>"], "r"))
     client = DataAPIClient(get_api_endpoint_from_stage(args['<stage>']), args['<api_token>'])
+    framework = client.get_framework(framework_slug)['frameworks']
+    framework_kwargs = framework['frameworkAgreementDetails']
+    framework_kwargs['frameworkName'] = framework['name']
```

Comment:
I thought about it but I like the idea of passing in a completely flat set of kwargs to `.render()` rather than have to have the templates know about the structure of our API responses.

Also, it was more typing to add `frameworkAgreementDetails.xxx` to everything in the template.